packages/typespec-test/test/loadTest/tspconfig.yaml

```diff
@@ -23,6 +23,7 @@ options:
     generateMetadata: true
     generateTest: true
     azureSdkForJs: false
+    enableOperationGroup: true
```

Comment:
We have to enable the flag because there are duplicate operation name in diff interface.